js/gated-content/src/views/CategoriesListingPage.vue

```diff
@@ -75,17 +75,11 @@ export default {
   },
   data() {
     return {
-      DEFAULT_SORT: 'weight_asc',
+      DEFAULT_SORT: 'date_asc',
```

Comment:
Please add a comment in task with info that `weight_asc` was removed from categories page

Reply:
Nevermind